bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -808,6 +808,8 @@ spec:
                   value: quay.io/konveyor/volume-snapshot-mover:latest
                 - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
                   value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
+                - name: RELATED_DATA_MOVER_DUMMY_POD_IMAGE
```

Comment:
Replace `RELATED_DATA_MOVER_DUMMY_POD_IMAGE` with `RELATED_IMAGE_DATA_MOVER_DUMMY_POD`

Reply:
Fixed. Thank you for the catch!